CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ cmake_minimum_required(VERSION 3.14)
 ##########################################################################
 
 project(MPIDiff
-        LANGUAGES CXX
+        LANGUAGES C CXX
```

Comment:
Does BLT require C support?

Reply:
or one of the tests or examples?

Reply:
Yeah, I think so. I get CMake warnings/errors about it in the HIP builds if I don't enable the C language.